gatorgrouper_django/gatorgrouper/templates/gatorgrouper/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Hi</title>
+  </head>
+  <body>
+    Hi
+
+    asdfadf
```

Comment:
This file should have some reasonable HTML stubs.

Reply:
The add to @Michionlion's suggestion, it might be easiest to delete index.html and merge in #143 to get the design team's template drafts